scripts/install_third_party.py

```diff
@@ -93,11 +93,13 @@ def download_files(source_url_root, target_dir, source_filenames):
     common.ensure_directory_exists(target_dir)
     for filename in source_filenames:
         if not os.path.exists(os.path.join(target_dir, filename)):
-            print 'Downloading file %s to %s' % (filename, target_dir)
+            print 'Downloading file %s to %s ...' % (filename, target_dir)
```

Comment:
Just a nit -- do we require this extra space between `%s` and `...`? Ditto below.

Reply:
Good question -- I added it because I didn't want the ellipsis to be considered part of the target_dir. But your call. I'm happy to change it if you want.

Reply:
Oh, then its completely fine. Let's keep it then. Thanks!